platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/AbstractTransformExecution.java

```diff
@@ -60,6 +60,8 @@
 
 abstract class AbstractTransformExecution implements UnitOfWork {
     private static final CachingDisabledReason NOT_CACHEABLE = new CachingDisabledReason(CachingDisabledReasonCategory.NOT_CACHEABLE, "Caching not enabled.");
+    private static final CachingDisabledReason CACHING_DISABLED_REASON = new CachingDisabledReason(CachingDisabledReasonCategory.NOT_CACHEABLE, "Caching disabled by property (experimental)");
```

Comment:
💅 Should this be `(internal)` instead of `(experimental)`?

Reply:
I'll change the message. Perhaps the message should state the full property that is used.

Reply:
Fixed 186eecb83b8b0a04e7a79807e36c36e4d3376409